lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
I see this code is supposed to be static so I guess that's the reason right? Normally in non-static code I would do something like

```ruby
class ActiveRecordFlorder
  attr_accessor :populate, :step, :min_delta

  def initialize  # set the default here
    @populate = false
    @min_delta = 0.0005
    @step = 2**16
  end
end
```